x-pack/legacy/plugins/index_management/public/app/components/template_form/template_form_schemas.tsx

```diff
@@ -68,14 +68,6 @@ export const schemas: Record<string, FormSchema> = {
             ),
           }),
         },
-        {
-          validator: startsWithField({
```

Comment:
I think we still want this validation when **creating** an index template. By deleting this code, I think it will remove it from both creating and editing. I reached out to @sebelga to see the best way to approach this with the [form lib](https://github.com/elastic/kibana/pull/41658).

Reply:
So here we have multiple options

- Add a `skipValidation` prop to `UseField` (that would go along with the `enabled: false`
- Extract and export the `name` config from the schema. Then add it on the UseField without the validations

```js
import { schemas, nameConfig } from '../template_form_schemas';
const { validations, nameConfigWithoutValidations } = nameConfig;

// And then use that config when editing

<UseField
  path="name"
  componentProps={{
    idAria: name.idAria,
    ['data-test-subj']: name.testSubject,
    euiFieldProps: { disabled: isEditing },
  }}
  config={ isEditing ? nameConfigWithoutValidations : nameConfig }
/>
```

I would go with the second one as it is simpler and we don't need to touch the lib for now. It would be a nice enhancement though.